apps/atqr/api/src/app/infra/email/mailer.service.ts

```diff
@@ -34,7 +34,7 @@ export class Mailer implements OnApplicationBootstrap, IMailer {
   async sendMail(email: Email) {
     const messageData = {
       from: this.fromEmail,
-      to: email.playerEmail,
+      to: email,
```

Comment:
Incorrect Type, email refers to a Email Object,  and the to property expects a string with a Email Address